src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStep.java

```diff
@@ -89,6 +90,8 @@ public DescriptorImpl getDescriptor() {
         return (DescriptorImpl) super.getDescriptor();
     }
 
+
+    @Symbol("exwsallocatestep")
```

Comment:
isn't this what it would be by default? i.e. normally its generated off the class name,
you can use symbol to get a more human readable name sometimes people camelcase it,
`exWsAllocateStep` 

But as Oleg said you shouldn't need this

Reply:
Yeah I will correct all these stuff when I figure out what made the test wrong.

Reply:
Whoo that really worked ! Thanks so much (I was being stupid not scolling down to those specific exceptions ) Next step would be write some really useful test cases.

Reply:
no problem 👍

Reply:
I htink that the right name would have been:

https://github.com/jenkinsci/external-workspace-manager-plugin/blob/08073fc5c2878633f5129b6d583b83c5263274a7/src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStep.java#L120-L123

Reply:
Let's resolve this conversation in favor of https://github.com/jenkinsci/external-workspace-manager-plugin/pull/56#discussion_r257254664

Reply:
In light of https://github.com/jenkinsci/external-workspace-manager-plugin/pull/56#discussion_r257254664 I suggest we use `@Symbol(exwsAllocateConfiguration)` here.